src/components/editor/property/PropertyLabelInfo.jsx

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import shortid from 'shortid'
+import React from 'react'
+import { OverlayTrigger, Popover } from 'react-bootstrap'
+import PropTypes from 'prop-types'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faInfoCircle, faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons'
+
+import _ from 'lodash'
+
+const PropertyLabelInfo = (props) => {
+  const key = shortid.generate()
+  const property = props.propertyTemplate
+
+  const popover = (
+    <Popover id="popover-basic" title={property.label} >
+      {property.remark}
+    </Popover>
+  )
+
+  let url
+
+  try {
+    url = new URL(property.remark)
+  } catch {
+    // Ignore
+  }
+
+  const urlLabel = (
+    <a href={url} className="prop-remark" alt={property.remark} key={key}
+       target="_blank" rel="noopener noreferrer">
+      <FontAwesomeIcon className="info-icon" icon={faExternalLinkAlt} />
+    </a>
+  )
+
+  const toolTipLabel = (
+    <OverlayTrigger trigger={['hover', 'focus']} placement="right" overlay={popover} key={key} >
+      <FontAwesomeIcon className="info-icon" icon={faInfoCircle} />
+    </OverlayTrigger>
+  )
+
+  if (url !== undefined) {
```

Comment:
I'd really like to see this split into 2 subcomponents that each do one specific thing.  Then this could be

```jsx
return url ? (<ExternalLink url="url">) : (<Tooltip property="property">)
```